package.json

```diff
@@ -6,7 +6,9 @@
   "scripts": {
     "clockin": "node execute.js clockin",
     "clockout": "node execute.js clockout",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "scheduler:clockin": "pm2 start 'yarn clockin'",
```

Comment:
it means you separate clockIn and clockOut scheduler?
Why make it separate tho?

I think it's better to have single scheduler that serve clockIn and/or clockOut..

Reply:
oh alright, I think I just need add a script again, to run both scheduler? How it is?
or, do you prefer  I add a function that run both?

Reply:
in user's perspective, I personally or they could have clockIn and clockOut in timely manners, so better to have it in one scheduler. You could drop one off, if you won't to have automatically. Besides, different script run, will have more resources to allocate

Reply:
hmm agree, have this fixed in latest commit